play/src/test/resources/logback-test.xml

```diff
@@ -36,4 +36,4 @@
         <appender-ref ref="ASYNCSTDOUT" />
     </root>
 
-</configuration>
+</configuration>
```

Comment:
for some reason it marks "</configuration>" as a remove and "</configuration>" as an add. There is no newline. But the whole file is actually new in this module.